app/Providers/AuthServiceProvider.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Providers;
+
+use Illuminate\Foundation\Support\Providers\AuthServiceProvider as ServiceProvider;
+use Illuminate\Support\Facades\Gate;
+
+class AuthServiceProvider extends ServiceProvider
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing doc comment for class AuthServiceProvider](https://app.codacy.com/gh/BuildForSDG/Team-059-Product/pullRequest?prid=5478046)